Grigore-George-Mihai/sandbox_playground
# Add S3 integration for file uploads with CarrierWave

## Summary
- This PR integrates CarrierWave and fog-aws to enable file uploads to Amazon S3. 
- It configures the application to use local storage in development and S3 in production.

## Related Issue(s)
- No

## Changes

## Description of Changes
- CarrierWave Configuration: Set up CarrierWave to handle file uploads, using local storage in development and S3 in production.
- S3 Integration: Configured fog-aws for communication with S3, including handling credentials via environment variables.

## How to Test
Steps to test the changes introduced in this PR (e.g., commands to run, environment setup, etc.).

1. In development, verify files are stored locally by uploading through the UI.
2. In production, ensure files are uploaded to the specified S3 bucket.
3. Run and check Rspec

## Screenshots (Optional)
If applicable, include screenshots to help explain your changes visually.

## Checklist
- [x] I have run `rubocop` and fixed any linter warnings.
- [x] I have run the test suite to ensure everything is passing.

## Additional Context (Optional)
Any other relevant information or context.
